charts/apiclarity/templates/fuzzer-template-configmap.yaml

```diff
@@ -0,0 +1,59 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ include "apiclarity.name" . }}-fuzzer-template
+  namespace: '{{ .Release.Namespace }}'
+  labels:
+    {{ include "apiclarity.labels" . }}
+data:
+  config: |-
+    apiVersion: batch/v1
+    kind: Job
+    metadata:
+      namespace: '{{ .Release.Namespace }}'
+      labels:
+{{- toYaml (index .Values "apiclarity" "labels") | nindent 8 }}
+    spec:
+      backoffLimit: 0
+      ttlsecondsafterfinished: 300
+      template:
+        metadata:
+          labels:
+{{- toYaml (index .Values "apiclarity" "labels") | nindent 12 }}
+        spec:
+          restartpolicy: Never
+          volumes:
+          - name: tmp-volume
+            emptyDir: {}
+          securityContext:
+            capabilities:
+              drop:
+              - all
+            runAsNonRoot: true
+            runAsGroup: 1001
+            runAsUser: 1001
+            privileged: false
+            allowPrivilegeEscalation: false
+          containers:
+          - name: vulnerability-scanner
+            image: '{{ index .Values "apiclarity" "modules" "fuzzer" "docker" "image" }}'
```

Comment:
you can use `{{ .Values.apiclarity.modules.fuzzer.docker.image }}`

Reply:
ok